apis/ml-core/ml-coreapi.yaml

```diff
@@ -0,0 +1,1290 @@
+openapi: 3.0.1
+info:
+  title: ML-Core Service API(s)
+  termsOfService: 'https://github.com/project-sunbird/sunbird-commons/blob/master/LICENSE'
+  description: >-
+    - The ML-Core Service is a centralised Service to support other services. Apis perform operations related to program entities user solutions etc 
+    
+    - The URL for ML-Core API(s) is `api`
+
+    - <b>Note:</b> These resources can be used in other services 
+  contact:
+    email: info@sunbird.org
+  version: '1.0'
+servers:
+- url: https://staging.open-sunbird.org/api
+  variables: {}
+paths: 
+  /users/mlcore/v1/programs:
+    post:
+      summary: List of targeted user programs 
+      description: |-
+        This API is used to get list of targeted user programs
+          - The endpoint for **User Programs** is `/users/mlcore/v1/programs`
+          - It is mandatory to provide values for parameters marked with *. 
+          - Mandatory fields cannot be null or empty.
+      tags:
+       - User APIs
+      operationId: List of targeted user programs
+      parameters:
+        - name: X-Authenticated-User-Token
+          in: header
+          description: Represents the authorized User ID to execute the API.
+          required: true
+          schema:
+            type: string
+        - name: Authorization
+          in: header
+          description: >-
+            To make use of the API, you require authorization. Raise a request
+            to the administrator for the use of the API. You will receive the
+            authorization key. Specify the key received, here.
+          required: true
+          schema:
+            type: string
+        - name: page
+          in: query
+          description: Page Number
+          schema:
+            type: string
+          example: 2
+        - name: limit
+          in: query
+          description: Number of objects to be returned
```

Comment:
Are these the total number of records returned or the number of records returned per page?